commands/team_test.go

```diff
@@ -108,3 +108,138 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
 		s.Require().Len(printer.GetLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestAddUsersCmd() {
+	mockTeam := model.Team{
+		Id:          "TeamId",
+		Name:        "team1",
+		DisplayName: "DisplayName",
+	}
+	mockUser := model.User{
+		Id:       "UserID",
+		Username: "ExampleUser",
+		Email:    "example@example.com",
+	}
+
+	s.Run("Add users with not enough arguments returns error", func() {
+
+		cmd := &cobra.Command{}
+		err := addUsersCmdF(s.client, cmd, []string{})
+
+		s.Require().Equal(err, errors.New("Not enough arguments."))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Add users with a team that cannot be found returns error", func() {
+
+		cmd := &cobra.Command{}
+
+		s.client.
+			EXPECT().
+			GetTeam("team1", "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName("team1", "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := addUsersCmdF(s.client, cmd, []string{"team1", "user1"})
+		s.Require().Equal(err, errors.New("Unable to find team 'team1'"))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Add users with nonexistent user in arguments prints error", func() {
+		printer.Clean()
+		cmd := &cobra.Command{}
+
+		s.client.
+			EXPECT().
+			GetTeam("team1", "").
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail("user1", "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByUsername("user1", "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUser("user1", "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := addUsersCmdF(s.client, cmd, []string{"team1", "user1"})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 1)
+		s.Require().Equal(printer.GetErrorLines()[0], "Can't find user 'user1'")
+	})
+
+	s.Run("Add users should print error when cannot add team member", func() {
+		printer.Clean()
+		cmd := &cobra.Command{}
+
+		s.client.
+			EXPECT().
+			GetTeam("team1", "").
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail("user1", "").
+			Return(&mockUser, &model.Response{Error: nil}).
+			Times(1)
+
+		mockError := &model.AppError{Message: "Cannot add team member"}
+
+		s.client.
+			EXPECT().
+			AddTeamMember("TeamId", "UserID").
+			Return(nil, &model.Response{Error: mockError}).
+			Times(1)
+
+		err := addUsersCmdF(s.client, cmd, []string{"team1", "user1"})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 1)
+		s.Require().Equal(printer.GetErrorLines()[0], "Unable to add 'user1' to team1. Error: : Cannot add team member, ")
```

Comment:
I would expect to create the mockError with `Where` and `DetailedError` fields to have here a more meaningful message. Not super important, but I prefer to have tests that looks as expected.